src/NUnitFramework/framework/Internal/Execution/EventPump.cs

```diff
@@ -142,14 +166,14 @@ public void Stop()
         /// </summary>
         private void PumpThreadProc()
         {
-            Log.Debug("Starting EventPump");
+            Log.Debug($"Starting EventPump {Name}");
```

Comment:
This would duplicate the word `EventPump` as it is part of `Name`

Reply:
Thanks!